utilities/convert_frosst/smirnoff99Frosst.ffxml

```diff
@@ -141,75 +144,75 @@
     <Improper smirks="[*:1]~[#6X3:2](=[#7X2,#7X3+1:3])~[#7:4]" id="i4" k1="10.5" periodicity1="2" phase1="180."/>
     <Proper smirks="[*:1]~[*:2]~[*:3]~[*:4]" id="t1" idivf1="4" k1="3.50" periodicity1="2" phase1="180.0"/>
     <Proper smirks="[*:1]-[#6X4:2]-[#6X4:3]-[*:4]" id="t2" idivf1="1" k1="0.156" periodicity1="3" phase1="0.0"/>
-    <Proper smirks="[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]" id="t3" idivf1="1" k1="0.180" periodicity1="3" phase1="0.0" k2="0.250" idivf2="1" periodicity2="2" phase2="180.0" phase3="180.0" k3="0.200" idivf3="1" periodicity3="1"/>
+    <Proper smirks="[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]" id="t3" idivf1="1" k1="0.180" periodicity1="3" phase1="0.0" periodicity2="2" idivf2="1" k2="0.250" phase2="180.0" idivf3="1" k3="0.200" periodicity3="1" phase3="180.0"/>
```

Comment:
@davidlmobley I've seen this come up in PRs before, there isn't actually a change to these torsion parameters, but the order of things in `ffxml` entry has changed. Is there a way to prevent this from happening so that we don't have a record of so many unnecessary changes in every pull request?

Reply:
You're welcome to look into the XML library. I would guess it's sort of like a python dictionary -- since the order doesn't matter, very little attention is paid into whether the order is preserved.

Reply:
OK that makes sense. At this point I think we'll continue making hand tuned changes with the smirnoffishFrcmod file so it doesn't matter too much, but might be worth looking at how parameters are written out for future tracking of changes. You're probably right about it using a dictionary for the underlying structure, I'll take a look at the XML library just in case.